--- badCRLBadDateHour.crl.raw.old	2012-02-08 12:52:54.923478429 -0500
+++ badCRLBadDateHour.crl.raw	2012-02-08 12:53:36.890645936 -0500
@@ -3,7 +3,7 @@
             nul 
         seq set seq oid 2.5.4.3 /* id_commonName */
                     prt "CRLBadDateHour"
-        utc "110411185728Z"
+        utc "110411245728Z"
         utc "460515185928Z"
         ctx+0x20
             seq seq oid 2.5.29.20 /* id_cRLNumber */
