--- badCertSIAMissing.raw.old	2011-09-19 16:57:17.000000000 -0400
+++ badCertSIAMissing.raw	2011-09-19 16:59:12.000000000 -0400
@@ -52,15 +52,6 @@
                     ocw seq seq oid 1.3.6.1.5.5.7.48.2 /* id_ad_caIssuers */
                                 ctx+0x06
                                     "rsync://rpki.bbn.com/conformance/root.cer"
-                seq oid 1.3.6.1.5.5.7.1.11 /* id_pe_subjectInfoAccess */
-                    ocw seq seq oid 1.3.6.1.5.5.7.48.5 /* id_ad_caRepository */
-                                ctx+0x06
-                                    "rsync://rpki.bbn.com/conformance/root/empt"
-                                    "y/"
-                            seq oid 1.3.6.1.5.5.7.48.10 /* id_ad_rpkiManifest */
-                                ctx+0x06
-                                    "rsync://rpki.bbn.com/conformance/root/empt"
-                                    "y/doesNotExist.mft"
                 seq oid 2.5.29.32 /* id_certificatePolicies */
                     boo 0xFF
                     ocw seq seq oid 1.3.6.1.5.5.7.14.2 /* id_pkix_rescerts_policy */
