--- badCertIssuerIa5.raw.old	2011-05-25 14:42:58.000000000 -0400
+++ badCertIssuerIa5.raw	2011-05-25 14:43:16.000000000 -0400
@@ -4,7 +4,7 @@
         seq oid 1.2.840.113549.1.1.11 /* id_sha_256WithRSAEncryption */
             nul 
         seq set seq oid 2.5.4.3 /* id_commonName */
-                    prt "root"
+                    ia5 "root"
         seq utc "110411185728Z"
             utc "460515185928Z"
         seq set seq oid 2.5.4.3 /* id_commonName */
