--- badCRLSigAlgOuter.raw.old	2011-06-07 13:13:41.000000000 -0400
+++ badCRLSigAlgOuter.raw	2011-06-07 13:13:52.000000000 -0400
@@ -11,7 +11,7 @@
                 seq oid 2.5.29.35  /* id_authKeyId */
                     oct 0x30168014E856567DF0F4FC5AF0B5EEA0 /* 0....VV}...Z.... */
                         0x2CB539D04ED0A2E7 /* ,.9.N... */
-    seq oid 1.2.840.113549.1.1.11  /* id_sha_256WithRSAEncryption */
+    seq oid 1.2.840.113549.1.1.12  /* id_sha_256WithRSAEncryption */
         nul 
     bit 0x003242855631C08DFA739C66B75DD25A /* .2B.V1...s.f.].Z */
         0x05240DF68196100AFE117196781A6C2E /* .$........q.x.l. */
