--- badCMSVersion4.roa.raw.old	2011-05-26 17:58:48.000000000 -0400
+++ badCMSVersion4.roa.raw	2011-05-26 17:59:08.000000000 -0400
@@ -1,6 +1,6 @@
 seq oid 1.2.840.113549.1.7.2 /* id_signedData */
     ctx+0x20
-        seq int 0x03 /* 3 */ 
+        seq int 0x04 /* 3 */ 
             set seq oid 2.16.840.1.101.3.4.2.1 /* id_sha256 */
                     nul 
             seq oid 1.2.840.113549.1.9.16.1.24 /* id_routeOriginAttestation */
