--- badCRLBadDateMonth0.crl.raw.old	2012-02-07 11:12:30.670924215 -0500
+++ badCRLBadDateMonth0.crl.raw	2012-02-07 11:12:42.778262100 -0500
@@ -3,7 +3,7 @@
             nul 
         seq set seq oid 2.5.4.3 /* id_commonName */
                     prt "CRLBadDateMonth0"
-        utc "110411185728Z"
+        utc "110011185728Z"
         utc "460515185928Z"
         ctx+0x20
             seq seq oid 2.5.29.20 /* id_cRLNumber */
