--- badCRLDeltaCRLInd.raw.old	2011-06-07 13:35:57.000000000 -0400
+++ badCRLDeltaCRLInd.raw	2011-06-07 13:38:03.000000000 -0400
@@ -12,6 +12,8 @@
                     ocw seq ctx+0x00
                                 0xE856567DF0F4FC5AF0B5EEA02CB539D0
                                 0x4ED0A2E7
+                seq oid 2.5.29.27
+                    ocw int 2
     seq oid 1.2.840.113549.1.1.11 /* id_sha_256WithRSAEncryption */
         nul 
     bit 0x00B8442FADBE72E2EE52451A1FCA1B2BACA25CC984185F551CB3D135FDF1FA1A
