--- badCertValToTyp.raw.old	2011-09-19 17:19:06.000000000 -0400
+++ badCertValToTyp.raw	2011-09-19 17:19:40.000000000 -0400
@@ -6,7 +6,7 @@
         seq set seq oid 2.5.4.3 /* id_commonName */
                     prt "root"
         seq utc "110411185728Z"
-            utc "460515185928Z"
+            gen "20460515185928Z"
         seq set seq oid 2.5.4.3 /* id_commonName */
                     prt "badCertValToTyp"
         seq seq oid 1.2.840.113549.1.1.1 /* id_rsadsi_rsaEncryption */
