--- badCertNoSKI.raw.old	2011-06-21 15:18:34.000000000 -0400
+++ badCertNoSKI.raw	2011-06-21 15:18:52.000000000 -0400
@@ -32,9 +32,6 @@
             seq seq oid 2.5.29.19 /* id_basicConstraints */
                     boo 0xFF
                     ocw seq boo 0xFF
-                seq oid 2.5.29.14 /* id_subjectKeyIdentifier */
-                    ocw oct 0xE3A5B96E6D0AC790FECC28C625CCE8D5
-                            0x26417857
                 seq oid 2.5.29.35 /* id_authKeyId */
                     ocw seq ctx+0x00
                                 0xD27F6E783B2B5CE535940D54C0A6F38F
