--- badCRLThisUpdateFuture.crl.raw.old	2011-12-06 15:03:11.607965363 -0500
+++ badCRLThisUpdateFuture.crl.raw	2011-12-06 15:03:22.705273684 -0500
@@ -3,7 +3,7 @@
             nul 
         seq set seq oid 2.5.4.3 /* id_commonName */
                     prt "CRLThisUpdateFuture"
-        utc "110411185728Z"
+        utc "410411185728Z"
         utc "460515185928Z"
         ctx+0x20
             seq seq oid 2.5.29.20 /* id_cRLNumber */
