--- badCRLBadDateSec.crl.raw.old	2012-01-17 11:09:22.234557490 -0500
+++ badCRLBadDateSec.crl.raw	2012-01-17 11:09:41.965119066 -0500
@@ -3,7 +3,7 @@
             nul 
         seq set seq oid 2.5.4.3 /* id_commonName */
                     prt "CRLBadDateSec"
-        utc "110322185728Z"
+        utc "110322185760Z"
         utc "460407185928Z"
         ctx+0x20
             seq seq oid 2.5.29.20 /* id_cRLNumber */
