--- badCRLBadDateSec.crl.raw.old	2012-02-07 11:07:12.162060086 -0500
+++ badCRLBadDateSec.crl.raw	2012-02-07 11:07:42.897915432 -0500
@@ -3,7 +3,7 @@
             nul 
         seq set seq oid 2.5.4.3 /* id_commonName */
                     prt "CRLBadDateSec"
-        utc "110411185728Z"
+        utc "110411185760Z"
         utc "460515185928Z"
         ctx+0x20
             seq seq oid 2.5.29.20 /* id_cRLNumber */
