--- badCMSSigInfoAttrs2SigTime.raw.old	2011-11-09 17:09:08.826610415 -0500
+++ badCMSSigInfoAttrs2SigTime.raw	2011-11-09 17:10:20.959595922 -0500
@@ -116,6 +116,10 @@
                         seq oid 1.2.840.113549.1.9.4 /* id_messageDigestAttr */
                             set oct 0x986E8A564EE4C9A4078DB7336E63D3B4
                                     0x25CF8ED649BF1E425CA765083E0493A7
+                        seq oid 1.2.840.113549.1.9.5 /* id_signingTimeAttr */
+                            set utc "110525220348Z"
+                        seq oid 1.2.840.113549.1.9.5 /* id_signingTimeAttr */
+                            set utc "110525220348Z"
                     seq oid 1.2.840.113549.1.1.11 /* id_sha_256WithRSAEncryption */
                         nul 
                     oct 0x063BBA346AA1C0E4C7543CA43E6A0A49
