--- badCMSSigInfoAttrsContTypeOid.raw.old	2011-11-09 15:49:25.291952502 -0500
+++ badCMSSigInfoAttrsContTypeOid.raw	2011-11-09 15:49:57.026832151 -0500
@@ -111,7 +111,7 @@
                     seq oid 2.16.840.1.101.3.4.2.1 /* id_sha256 */
                         nul 
                     ctx+0x20
-                        seq oid 1.2.840.113549.1.9.3 /* id_contentTypeAttr */
+                        seq oid 1.2.840.113549.1.9.6 /* id_somethingElse */
                             set oid 1.2.840.113549.1.9.16.1.24 /* id_routeOriginAttestation */
                         seq oid 1.2.840.113549.1.9.4 /* id_messageDigestAttr */
                             set oct 0x986E8A564EE4C9A4078DB7336E63D3B4
