--- badCRLEntryNoSerNum.raw.old	2011-06-07 13:39:25.000000000 -0400
+++ badCRLEntryNoSerNum.raw	2011-06-07 13:41:00.000000000 -0400
@@ -5,6 +5,7 @@
                     prt "CRLEntryNoSerNum"
         utc "110411185728Z"
         utc "460515185928Z"
+        seq seq utc "110301010101Z"
         ctx+0x20
             seq seq oid 2.5.29.20 /* id_cRLNumber */
                     ocw int 0x01 /* 1 */ 
