--- badCRLDeltaCRLInd.crl.raw.old	2011-09-29 18:21:49.000000000 -0400
+++ badCRLDeltaCRLInd.crl.raw	2011-09-29 18:22:51.000000000 -0400
@@ -12,6 +12,8 @@
                     ocw seq ctx+0x00
                                 0xD27F6E783B2B5CE535940D54C0A6F38F
                                 0x5B7804D4
+                seq oid 2.5.29.27 /* id_deltaCRLIndicator */
+                    ocw int 2
     seq oid 1.2.840.113549.1.1.11 /* id_sha_256WithRSAEncryption */
         nul 
     bit 0x0059AD5907AB88F88E13BD2A162D14F430CFAB002BCC6B8FFAABF5925A66A8D9
