--- badCMSSigInfoAttrsNoMsgDigest.raw.old	2011-11-09 16:58:29.698022783 -0500
+++ badCMSSigInfoAttrsNoMsgDigest.raw	2011-11-09 16:58:42.789382931 -0500
@@ -113,9 +113,6 @@
                     ctx+0x20
                         seq oid 1.2.840.113549.1.9.3 /* id_contentTypeAttr */
                             set oid 1.2.840.113549.1.9.16.1.24 /* id_routeOriginAttestation */
-                        seq oid 1.2.840.113549.1.9.4 /* id_messageDigestAttr */
-                            set oct 0x986E8A564EE4C9A4078DB7336E63D3B4
-                                    0x25CF8ED649BF1E425CA765083E0493A7
                     seq oid 1.2.840.113549.1.1.11 /* id_sha_256WithRSAEncryption */
                         nul 
                     oct 0x2CC2C8B6DC9AE25B0EE9899D2A108B99
