--- badCRLIssAltName.crl.raw.old	2011-11-12 04:44:55.195791715 -0500
+++ badCRLIssAltName.crl.raw	2011-11-12 04:45:36.498410341 -0500
@@ -12,6 +12,8 @@
                     ocw seq ctx+0x00
                                 0xE856567DF0F4FC5AF0B5EEA02CB539D0
                                 0x4ED0A2E7
+                seq oid 2.5.29.18
+                    ocw seq ctx+0x01 "alt name"
     seq oid 1.2.840.113549.1.1.11 /* id_sha_256WithRSAEncryption */
         nul 
     bit 0x008A05D66F32C4D046118BA06BCE475B0ED2549A9351DD3196A652DDFABB52CC
