--- badCert2CRLDP.raw.old	2011-09-19 17:38:49.000000000 -0400
+++ badCert2CRLDP.raw	2011-09-19 17:41:17.000000000 -0400
@@ -75,6 +75,12 @@
                     ocw seq ctx+0x20
                                 seq seq int 0x01 /* 1 */ 
                                         int 0x0100 /* 256 */ 
+                seq oid 2.5.29.31 /* id_cRLDistributionPoints */
+                    ocw seq seq ctx+0x20
+                                    ctx+0x20
+                                        ctx+0x06
+                                            "rsync://rpki.bbn.com/conformance/r"
+                                            "oot/root.crl"
     seq oid 1.2.840.113549.1.1.11 /* id_sha_256WithRSAEncryption */
         nul 
     bit 0x00910EF051F3145B530E9AA827C38273D242F69876B55BB00090C5E65B345E94
