--- badCRLBadDateMonth13.crl.raw.old	2012-01-17 11:23:11.643150671 -0500
+++ badCRLBadDateMonth13.crl.raw	2012-01-17 11:23:21.596429205 -0500
@@ -3,7 +3,7 @@
             nul 
         seq set seq oid 2.5.4.3 /* id_commonName */
                     prt "CRLBadDateMonth13"
-        utc "110322185728Z"
+        utc "111322185728Z"
         utc "460407185928Z"
         ctx+0x20
             seq seq oid 2.5.29.20 /* id_cRLNumber */
