--- badCertIssuer2Sets.raw.old	2011-05-25 14:40:48.000000000 -0400
+++ badCertIssuer2Sets.raw	2011-05-25 14:42:12.000000000 -0400
@@ -5,6 +5,8 @@
             nul 
         seq set seq oid 2.5.4.3 /* id_commonName */
                     prt "root"
+            set seq oid 2.5.4.5 /* id_commonName */
+                    prt "5"
         seq utc "110411185728Z"
             utc "460515185928Z"
         seq set seq oid 2.5.4.3 /* id_commonName */
