--- badCertNoAKI.raw.old	2011-06-21 15:22:06.000000000 -0400
+++ badCertNoAKI.raw	2011-06-21 15:22:22.000000000 -0400
@@ -35,10 +35,6 @@
                 seq oid 2.5.29.14 /* id_subjectKeyIdentifier */
                     ocw oct 0xE3A5B96E6D0AC790FECC28C625CCE8D5
                             0x26417857
-                seq oid 2.5.29.35 /* id_authKeyId */
-                    ocw seq ctx+0x00
-                                0xD27F6E783B2B5CE535940D54C0A6F38F
-                                0x5B7804D4
                 seq oid 2.5.29.15 /* id_keyUsage */
                     boo 0xFF
                     ocw bit 0x0106
