--- badCMSNoDigestAlgs.roa.raw.old	2011-05-26 18:03:01.000000000 -0400
+++ badCMSNoDigestAlgs.roa.raw	2011-05-26 18:03:14.000000000 -0400
@@ -1,8 +1,6 @@
 seq oid 1.2.840.113549.1.7.2 /* id_signedData */
     ctx+0x20
         seq int 0x03 /* 3 */ 
-            set seq oid 2.16.840.1.101.3.4.2.1 /* id_sha256 */
-                    nul 
             seq oid 1.2.840.113549.1.9.16.1.24 /* id_routeOriginAttestation */
                 ctx+0x20
                     ocw seq int 0x08 /* 8 */ 
