--- badCertSubject2Seq.raw.old	2011-09-19 17:05:50.000000000 -0400
+++ badCertSubject2Seq.raw	2011-09-19 17:06:59.000000000 -0400
@@ -9,6 +9,8 @@
             utc "460515185928Z"
         seq set seq oid 2.5.4.3 /* id_commonName */
                     prt "badCertSubject2Seq"
+            set seq oid 2.5.4.3 /* id_commonName */
+                    prt "badCertSubject2Seq"
         seq seq oid 1.2.840.113549.1.1.1 /* id_rsadsi_rsaEncryption */
                 nul 
             bit 0x003082010A0282010100BFBD06FD096D
