--- badCRLNoAKI.crl.raw.old	2011-11-12 11:48:48.449719357 -0500
+++ badCRLNoAKI.crl.raw	2011-11-12 11:49:12.184094934 -0500
@@ -8,10 +8,6 @@
         ctx+0x20
             seq seq oid 2.5.29.20 /* id_cRLNumber */
                     ocw int 0x01 /* 1 */ 
-                seq oid 2.5.29.35 /* id_authKeyId */
-                    ocw seq ctx+0x00
-                                0xB3EC8D4740D4C0342816C63565DAE2B6
-                                0x39A01D3D
     seq oid 1.2.840.113549.1.1.11 /* id_sha_256WithRSAEncryption */
         nul 
     bit 0x00731DE66786B5C93AC779E14829061E00AE4C375279944B379315DA332A7DAB
