--- badCRLSigAlgOuter.crl.raw.old	2011-12-12 20:17:58.567515679 -0500
+++ badCRLSigAlgOuter.crl.raw	2011-12-12 20:18:39.036656089 -0500
@@ -12,7 +12,7 @@
                     ocw seq ctx+0x00
                                 0xC032A174C05663781981783C78EABB54
                                 0x476CD8DF
-    seq oid 1.2.840.113549.1.1.11 /* id_sha_256WithRSAEncryption */
+    seq oid 1.2.840.113549.1.1.12 /* id_sha_384WithRSAEncryption */
         nul 
     bit 0x00868A2316FB3AECA6B0540F14B0F4313C1D27938CE88683AFD90DC1177FC3E2
         0xC1B2EE0C8FBCE1C8CE7111472EC2E469C0C3ADDBF580B29E4C058718B288D029
