--- badCMSSigInfoNoAttrs.raw.old	2011-12-01 17:36:11.000000000 -0500
+++ badCMSSigInfoNoAttrs.raw	2011-12-01 17:36:51.000000000 -0500
@@ -111,11 +111,5 @@
                     seq oid 2.16.840.1.101.3.4.2.1 /* id_sha256 */
                         nul 
-                    ctx+0x20
-                        seq oid 1.2.840.113549.1.9.3 /* id_contentTypeAttr */
-                            set oid 1.2.840.113549.1.9.16.1.24 /* id_routeOriginAttestation */
-                        seq oid 1.2.840.113549.1.9.4 /* id_messageDigestAttr */
-                            set oct 0x5808AD5429D14BE2CD5CD9B912B61FD0
-                                    0x234CB1D2DD939DB9BC65DAC7014366E4
                     seq oid 1.2.840.113549.1.1.11 /* id_sha_256WithRSAEncryption */
                         nul 
                     oct 0x0ACC37BD4A6F652A54AC11711AE19DFA
