--- badCertNoBasicConstr.raw.old	2011-05-25 15:36:08.000000000 -0400
+++ badCertNoBasicConstr.raw	2011-05-25 15:37:15.000000000 -0400
@@ -29,10 +29,7 @@
                 0xC606A948A9C5B0EAAAA7051DF03E49BB
                 0x8395A87C2F1C516397470203010001
         ctx+0x23
-            seq seq oid 2.5.29.19 /* id_basicConstraints */
-                    boo 0xFF
-                    ocw seq boo 0xFF
-                seq oid 2.5.29.14 /* id_subjectKeyIdentifier */
+            seq seq oid 2.5.29.14 /* id_subjectKeyIdentifier */
                     ocw oct 0xD8C8131BF55527DA27C425E86006185A
                             0x6F36A189
                 seq oid 2.5.29.35 /* id_authKeyId */
