--- badCRLIssAltName.crl.raw.old	2011-11-12 11:43:51.149657357 -0500
+++ badCRLIssAltName.crl.raw	2011-11-12 11:45:04.876936983 -0500
@@ -6,7 +6,9 @@
         utc "110411185728Z"
         utc "460515185928Z"
         ctx+0x20
-            seq seq oid 2.5.29.20 /* id_cRLNumber */
+            seq seq oid 2.5.29.18
+                    ocw seq ctx+0x01 "alt name"
+                seq oid 2.5.29.20 /* id_cRLNumber */
                     ocw int 0x01 /* 1 */ 
                 seq oid 2.5.29.35 /* id_authKeyId */
                     ocw seq ctx+0x00
