--- badCRLIssDistPt.crl.raw.old	2011-09-29 18:16:32.000000000 -0400
+++ badCRLIssDistPt.crl.raw	2011-09-29 18:18:44.000000000 -0400
@@ -12,6 +12,10 @@
                     ocw seq ctx+0x00
                                 0xD27F6E783B2B5CE535940D54C0A6F38F
                                 0x5B7804D4
+                seq oid 2.5.29.28 /* id_issuingDistributionPoint */
+                    ocw seq ctx+0x20
+                                ctx+0x20
+                                    ctx+0x01 "some name"
     seq oid 1.2.840.113549.1.1.11 /* id_sha_256WithRSAEncryption */
         nul 
     bit 0x00B9601FE7C1DAA0956C7DCC9762B31277FAD10DF2314C18CEC6DEC4242DCB59
