--- badCertInnerSigAlg.raw.old	2011-05-25 14:37:34.000000000 -0400
+++ badCertInnerSigAlg.raw	2011-05-25 14:38:20.000000000 -0400
@@ -1,7 +1,7 @@
 seq seq ctx+0x20
             int 0x02 /* 2 */ 
         int 0x78 /* 120 */ 
-        seq oid 1.2.840.113549.1.1.11 /* id_sha_256WithRSAEncryption */
+        seq oid 1.2.840.113549.1.1.12 /* id_sha_256WithRSAEncryption */
             nul 
         seq set seq oid 2.5.4.3 /* id_commonName */
                     prt "root"
