--- badCMSSigInfoAttrs2BinSigTime.roa.raw.old	2011-05-25 18:05:03.000000000 -0400
+++ badCMSSigInfoAttrs2BinSigTime.roa.raw	2011-05-25 18:07:46.000000000 -0400
@@ -117,11 +117,15 @@
                     ctx+0x20
                         seq oid 1.2.840.113549.1.9.3 /* id_contentTypeAttr */
                             set oid 1.2.840.113549.1.9.16.1.24 /* id_routeOriginAttestation */
+                        seq oid 1.2.840.113549.1.9.16.2.46
+                            int 0x50000000
                         seq oid 1.2.840.113549.1.9.5 /* id_signingTimeAttr */
                             set utc "110525220502Z"
                         seq oid 1.2.840.113549.1.9.4 /* id_messageDigestAttr */
                             set oct 0x986E8A564EE4C9A4078DB7336E63D3B4
                                     0x25CF8ED649BF1E425CA765083E0493A7
+                        seq oid 1.2.840.113549.1.9.16.2.46
+                            int 0x50000000
                     seq oid 1.2.840.113549.1.1.1 /* id_rsadsi_rsaEncryption */
                         nul 
                     oct 0x030E3FF6AE57228388473A6C4B3119A5
