--- badCertOuterSigAlg.raw.old	2011-05-25 14:53:59.000000000 -0400
+++ badCertOuterSigAlg.raw	2011-05-25 14:54:40.000000000 -0400
@@ -76,7 +76,7 @@
                     ocw seq ctx+0x20
                                 seq seq int 0x01 /* 1 */ 
                                         int 0x0100 /* 256 */ 
-    seq oid 1.2.840.113549.1.1.11 /* id_sha_256WithRSAEncryption */
+    seq oid 1.2.840.113549.1.1.12 /* id_sha_256WithRSAEncryption */
         nul 
     bit 0x006C562A805514144BFE8E6E66CB90675113077846EF81FE79D385D8B9308627
         0x1D22C17FE477D3EFB51F69C8745D429A142430D735184BA7AFAB3D73D097D571
