--- badCMSSigInfoNoHashAlg.roa.raw.old	2011-05-26 10:50:06.000000000 -0400
+++ badCMSSigInfoNoHashAlg.roa.raw	2011-05-26 10:50:56.000000000 -0400
@@ -112,8 +112,6 @@
                     ctx+0x00
                         0x7EF1B5892987BF7505BC6E683EB12D7B
                         0x02F27158
-                    seq oid 2.16.840.1.101.3.4.2.1 /* id_sha256 */
-                        nul 
                     ctx+0x20
                         seq oid 1.2.840.113549.1.9.3 /* id_contentTypeAttr */
                             set oid 1.2.840.113549.1.9.16.1.24 /* id_routeOriginAttestation */
