--- badMFTNextUpdUTF.raw.old	2011-06-06 14:21:30.000000000 -0400
+++ badMFTNextUpdUTF.raw	2011-06-06 14:21:54.000000000 -0400
@@ -7,7 +7,7 @@
                 ctx+0x20
                     ocw seq int 0x01 /* 1 */ 
                             gen "20110411185728Z"
-                            gen "20460515185928Z"
+                            utf "460515185928Z"
                             oid 2.16.840.1.101.3.4.2.1 /* id_sha256 */
                             seq seq ia5 "MFTNextUpdUTF.crl"
                                     bit 0x00763A60DD3B7B8600D3AC699E2D536A
