--- badMFTHashAlg.raw.old	2011-06-06 15:02:31.000000000 -0400
+++ badMFTHashAlg.raw	2011-06-06 15:02:46.000000000 -0400
@@ -8,7 +8,7 @@
                     ocw seq int 0x01 /* 1 */ 
                             gen "20110411185728Z"
                             gen "20460515185928Z"
-                            oid 2.16.840.1.101.3.4.2.1 /* id_sha256 */
+                            oid 2.16.840.1.101.3.4.2.2 /* id_sha256 */
                             seq seq ia5 "MFTHashAlg.crl"
                                     bit 0x0038E604782F4DECEF2A724E28DF3EBD
                                         0x770C4AE00CAB5CE21EBF1648A26EC54F
