--- badMFTHashOctetStr.raw.old	2011-11-12 05:09:32.818021039 -0500
+++ badMFTHashOctetStr.raw	2011-11-12 05:09:51.873022065 -0500
@@ -10,7 +10,7 @@
                             gen "20460515185928Z"
                             oid 2.16.840.1.101.3.4.2.1 /* id_sha256 */
                             seq seq ia5 "MFTHashOctetStr.crl"
-                                    bit 0x00A22E2C1A32463E06A893D23809EDC4
+                                    oct 0xA22E2C1A32463E06A893D23809EDC4
                                         0x6C23EC0F5CBB56D9F41C3295E241AB38
                                         0x0F
             ctx+0x20
