--- badMFTFileHashShort.raw.old	2011-11-12 05:04:15.388280152 -0500
+++ badMFTFileHashShort.raw	2011-11-12 05:04:23.193499972 -0500
@@ -12,7 +12,6 @@
                             seq seq ia5 "MFTFileHashShort.crl"
                                     bit 0x003043C757DB7E82D72D95BA418ED6EA
                                         0x2D14D2CC1C82FA060E315A4D4DC3BACA
-                                        0xC6
             ctx+0x20
                 seq seq ctx+0x20
                             int 0x02 /* 2 */ 
