--- badMFTFileHashLong.raw.old	2011-11-12 05:05:45.626008809 -0500
+++ badMFTFileHashLong.raw	2011-11-12 05:06:15.059010213 -0500
@@ -12,7 +12,7 @@
                             seq seq ia5 "MFTFileHashLong.crl"
                                     bit 0x005EE90D474B34F3BE6CA1E4DBF72664
                                         0xC73D31E719B5150164D9257C3127F9F7
-                                        0xA7
+                                        0xA742
             ctx+0x20
                 seq seq ctx+0x20
                             int 0x02 /* 2 */ 
