--- badMFTHashOctetStr.raw.old	2011-11-12 11:56:11.236332197 -0500
+++ badMFTHashOctetStr.raw	2011-11-12 11:57:08.811747281 -0500
@@ -10,7 +10,7 @@
                             gen "20460515185928Z"
                             oid 2.16.840.1.101.3.4.2.1 /* id_sha256 */
                             seq seq ia5 "MFTHashOctetStr.crl"
-                                    bit 0x004220474DCD46CB53A74ED46D34CEB2
+                                    oct 0x4220474DCD46CB53A74ED46D34CEB2
                                         0x8B085EABF1A6DBDDB5411E04D1CB7E5C
                                         0x85
             ctx+0x20
