--- badMFTHashAlg.raw.old	2012-04-30 17:02:45.547766240 -0400
+++ badMFTHashAlg.raw	2012-04-30 17:04:33.447761726 -0400
@@ -8,11 +8,10 @@
                     ocw seq int 0x01 /* 1 */ 
                             gen "20110411185728Z"
                             gen "20460515185928Z"
-                            oid 2.16.840.1.101.3.4.2.1 /* id_sha256 */
+                            oid 1.3.14.3.2.26 /* id_sha1 */
                             seq seq ia5 "MFTHashAlg.crl"
                                     bit 0x003CD09A6AF07166550CA0857311CBF6
-                                        0xC7F02D812874570E1E1D3B62554481BB
-                                        0x2D
+                                        0xC7F02D8128
             ctx+0x20
                 seq seq ctx+0x20
                             int 0x02 /* 2 */ 
