--- badROAIPMaxLth.raw.old	2012-02-11 04:51:11.870680692 -0500
+++ badROAIPMaxLth.raw	2012-02-11 04:51:25.910371418 -0500
@@ -8,6 +8,7 @@
                     ocw seq int 0x08 /* 8 */ 
                             seq seq oct 0x0001
                                     seq seq bit 0x000101
+                                            int 33
                                 seq oct 0x0002
                                     seq seq bit 0x00010201
             ctx+0x20
