--- badCMSSigInfoVersion.roa.raw.old	2011-05-26 18:08:59.000000000 -0400
+++ badCMSSigInfoVersion.roa.raw	2011-05-26 18:09:14.000000000 -0400
@@ -108,7 +108,7 @@
                         0xC016380414667B41F0D47DB99984E552
                         0x5E812458A27C6408A9C4001FA6CE132B
                         0x60
-            set seq int 0x03 /* 3 */ 
+            set seq int 0x02 /* 3 */ 
                     ctx+0x00
                         0x4E1E06BEAAED836D29B6DA65A6901EFE
                         0x3BF9B168
