--- badCMSSigInfoAttrs2MsgDigest.roa.raw.old	2011-05-25 18:02:18.000000000 -0400
+++ badCMSSigInfoAttrs2MsgDigest.roa.raw	2011-05-25 18:03:22.000000000 -0400
@@ -117,6 +117,9 @@
                     ctx+0x20
                         seq oid 1.2.840.113549.1.9.3 /* id_contentTypeAttr */
                             set oid 1.2.840.113549.1.9.16.1.24 /* id_routeOriginAttestation */
+                        seq oid 1.2.840.113549.1.9.4 /* id_messageDigestAttr */
+                            set oct 0x986E8A564EE4C9A4078DB7336E63D3B4
+                                    0x25CF8ED649BF1E425CA765083E0493A7
                         seq oid 1.2.840.113549.1.9.5 /* id_signingTimeAttr */
                             set utc "110525220218Z"
                         seq oid 1.2.840.113549.1.9.4 /* id_messageDigestAttr */
