--- badCMSSigInfoNoSid.roa.raw.old	2011-05-26 10:41:23.000000000 -0400
+++ badCMSSigInfoNoSid.roa.raw	2011-05-26 10:42:32.000000000 -0400
@@ -109,9 +109,6 @@
                         0xBA24DDF4750492808A286318EAFD6158
                         0x24
             set seq int 0x03 /* 3 */ 
-                    ctx+0x00
-                        0x97C38D2643AA342D69C871928CCEE793
-                        0x2C0D01A3
                     seq oid 2.16.840.1.101.3.4.2.1 /* id_sha256 */
                         nul 
                     ctx+0x20
