--- badCertSKILth.raw.old	2011-05-25 15:12:19.000000000 -0400
+++ badCertSKILth.raw	2011-05-25 15:13:02.000000000 -0400
@@ -34,7 +34,7 @@
                     ocw seq boo 0xFF
                 seq oid 2.5.29.14 /* id_subjectKeyIdentifier */
                     ocw oct 0xD8C8131BF55527DA27C425E86006185A
-                            0x6F36A189
+                            0x6F36A1
                 seq oid 2.5.29.35 /* id_authKeyId */
                     ocw seq ctx+0x00
                                 0x66692F54208FC6E03A1227B2DE8289E5
