--- badCMS2Certs.roa.raw.old	2011-05-26 17:55:50.000000000 -0400
+++ badCMS2Certs.roa.raw	2011-05-26 17:57:14.000000000 -0400
@@ -108,6 +108,103 @@
                         0x9BD81D40CD2AD3A15F41CC79C1A9C395
                         0x8A0187939C24A856293B6410C6EB8AA0
                         0x8D
+                seq seq ctx+0x20
+                            int 0x02 /* 2 */ 
+                        int 0x0202 /* 514 */ 
+                        seq oid 1.2.840.113549.1.1.11 /* id_sha_256WithRSAEncryption */
+                            nul 
+                        seq set seq oid 2.5.4.3 /* id_commonName */
+                                    prt "root"
+                        seq utc "110411185728Z"
+                            utc "460515185928Z"
+                        seq set seq oid 2.5.4.3 /* id_commonName */
+                                    prt "badCMSEE2Certs"
+                        seq seq oid 1.2.840.113549.1.1.1 /* id_rsadsi_rsaEncryption */
+                                nul 
+                            bit 0x003082010A0282010100E578C36AEE2B
+                                0x3F6B12392FB5F73D06DE1DA33138170A
+                                0xF97E3E7D798DF01A09A7B575409FC862
+                                0x45D5A6EF50618538D93DE36CF66E1214
+                                0x0DC6FA77534041D6AD271B37B85A7EF2
+                                0x95A7F556CCBAF7CC013ADBAE7BF16ED2
+                                0x296AFD5773662919D895F8804CB409FF
+                                0xD92D94DE56C530201274BB57F2A12C8D
+                                0xE2071539439329A947EBD3972711771A
+                                0xAD9D8B5E76AA94635CE138338E9814B2
+                                0x8F6B2780D71672698B6E787A4EB0CAD8
+                                0xAF3CE4564F9F9CC718F907496CB17223
+                                0x1D6F3060B046B51C94C8CE4FAFE7DF5A
+                                0xB8939A8FEFC74ED22A80DEC3E00B5F42
+                                0xB1B3BD8E836055C8D1948829C88B5B9E
+                                0xB85695B3EEA3EE9332358C70C12D9B5B
+                                0x86F9E25DD8BBBA1B318D0203010001
+                        ctx+0x23
+                            seq seq oid 2.5.29.14 /* id_subjectKeyIdentifier */
+                                    ocw oct 0x79FEA0C3F214BF00562CA74C39D0FBD4
+                                            0x16EAC6AB
+                                seq oid 2.5.29.35 /* id_authKeyId */
+                                    ocw seq ctx+0x00
+                                                0x66692F54208FC6E0
+                                                0x3A1227B2DE8289E5
+                                                0x3A3EA861
+                                seq oid 2.5.29.15 /* id_keyUsage */
+                                    boo 0xFF
+                                    ocw bit 0x0780
+                                seq oid 2.5.29.31 /* id_cRLDistributionPoints */
+                                    ocw seq seq ctx+0x20
+                                                    ctx+0x20
+                                                        ctx+0x06
+                                                            "rsync://rpki.bbn.c"
+                                                            "om/conformance/roo"
+                                                            "t"
+                                seq oid 1.3.6.1.5.5.7.1.1 /* id_pkix_authorityInfoAccess */
+                                    ocw seq seq oid 1.3.6.1.5.5.7.48.2 /* id_ad_caIssuers */
+                                                ctx+0x06
+                                                    "rsync://roa-pki.bbn.com/co"
+                                                    "nformance/root"
+                                seq oid 1.3.6.1.5.5.7.1.11 /* id_pe_subjectInfoAccess */
+                                    ocw seq seq oid 1.3.6.1.5.5.7.48.11 /* id_ad_signedObject */
+                                                ctx+0x06
+                                                    "rsync://rpki.bbn.com/confo"
+                                                    "rmance/root/good.roa"
+                                            seq oid 1.3.6.1.5.5.7.48.10 /* id_ad_rpkiManifest */
+                                                ctx+0x06
+                                                    "rsync://rpki.bbn.com/confo"
+                                                    "rmance/root/cert_syntax/1."
+                                                    "a.i.1.goodVersion.mft"
+                                seq oid 2.5.29.32 /* id_certificatePolicies */
+                                    boo 0xFF
+                                    ocw seq seq oid 1.3.6.1.5.5.7.14.2 /* id_pkix_rescerts_policy */
+                                seq oid 1.3.6.1.5.5.7.1.7 /* id_pe_ipAddrBlock */
+                                    boo 0xFF
+                                    ocw seq seq oct 0x0001
+                                                seq bit 0x000101
+                                            seq oct 0x0002
+                                                seq bit 0x00010201
+                                seq oid 1.3.6.1.5.5.7.1.8 /* id_pe_autonomousSysNum */
+                                    boo 0xFF
+                                    ocw seq ctx+0x20
+                                                seq seq int 0x01 /* 1 */ 
+                                                        int 0x0100 /* 256 */ 
+                    seq oid 1.2.840.113549.1.1.11 /* id_sha_256WithRSAEncryption */
+                        nul 
+                    bit 0x0018102C6FFE695154829C7B487D5E30
+                        0xF0DC157935AED254C1A2A848E2909C92
+                        0xED3374B2BD574EB3718E7ED22F54BD4C
+                        0x599452856E1571457019D437F91F1F55
+                        0x2A1EEDFC89BA4492C6A9AD749C7E20D2
+                        0x220F713313A331B9F1E4C3050C5EC62F
+                        0xDAB64B998004A85AD04DA6474FFEE569
+                        0x774EA4895BE09A016F8464F24CCE4CCD
+                        0xE0CE78E25F7EFC51DC691D2C22F4C90A
+                        0xCA752795E61CAEB03BB13C88702B7D29
+                        0x45996392641F29505F1A415936B971ED
+                        0xCD861DDD3F12A8B6E479EE147B0DFE0E
+                        0x34EFAFF9DBAEB4D4325FE0DCE6B1AB74
+                        0xF46828167837B8CE309EF3077627EA81
+                        0x9BD81D40CD2AD3A15F41CC79C1A9C395
+                        0x8A0187939C24A856293B6410C6EB8AA0
+                        0x8E
             set seq int 0x03 /* 3 */ 
                     ctx+0x00
                         0x79FEA0C3F214BF00562CA74C39D0FBD4
