--- badCertSubject2Seq.raw.old	2011-05-25 15:15:43.000000000 -0400
+++ badCertSubject2Seq.raw	2011-05-25 15:16:46.000000000 -0400
@@ -9,6 +9,8 @@
             utc "460515185928Z"
         seq set seq oid 2.5.4.3 /* id_commonName */
                     prt "badCertSubject2Seq"
+                seq oid 2.5.4.3 /* id_commonName */
+                    prt "badCertSubject2Seq"
         seq seq oid 1.2.840.113549.1.1.1 /* id_rsadsi_rsaEncryption */
                 nul 
             bit 0x003082010A0282010100C247EF57C065
