--- badCertPubKeyExp.raw.old	2011-05-25 14:56:03.000000000 -0400
+++ badCertPubKeyExp.raw	2011-05-25 14:56:50.000000000 -0400
@@ -11,7 +11,7 @@
                     prt "badCertPubKeyExp"
         seq seq oid 1.2.840.113549.1.1.1 /* id_rsadsi_rsaEncryption */
                 nul 
-            bit 0x003082010A0282010100C247EF57C065
+            bit 0x00308201080282010100C247EF57C065
                 0x453DAB3ECFC9D8432CB492D1765815EC
                 0xC907FDEE0500D195CAF8BC084ACE982B
                 0xF0C733EDDAFDB0E95AEE558931CD59FF
@@ -27,7 +27,7 @@
                 0xD5F58D2603D0A644A5BDADB705570204
                 0x81114A94E7916829DC94D94CDAB247A6
                 0xC606A948A9C5B0EAAAA7051DF03E49BB
-                0x8395A87C2F1C516397470203010001
+                0x8395A87C2F1C51639747020103
         ctx+0x23
             seq seq oid 2.5.29.19 /* id_basicConstraints */
                     boo 0xFF
