--- badROAFamily.raw.old	2011-05-25 18:29:00.000000000 -0400
+++ badROAFamily.raw	2011-05-25 18:29:20.000000000 -0400
@@ -8,7 +8,7 @@
                     ocw seq int 0x08 /* 8 */ 
                             seq seq oct 0x0001
                                     seq seq bit 0x0001
-                                seq oct 0x0002
+                                seq oct 0x0003
                                     seq seq bit 0x00010200
             ctx+0x20
                 seq seq ctx+0x20
