--- badCRLNextUpdatePast.crl.raw.old	2011-12-12 16:01:53.363368744 -0500
+++ badCRLNextUpdatePast.crl.raw	2011-12-12 16:02:10.815864510 -0500
@@ -3,8 +3,8 @@
             nul 
         seq set seq oid 2.5.4.3 /* id_commonName */
                     prt "CRLNextUpdatePast"
-        utc "110411185728Z"
-        utc "460515185928Z"
+        utc "050411185728Z"
+        utc "060515185928Z"
         ctx+0x20
             seq seq oid 2.5.29.20 /* id_cRLNumber */
                     ocw int 0x01 /* 1 */ 
