--- badCRLDeltaCRLInd.crl.raw.old	2011-11-12 04:49:54.265148822 -0500
+++ badCRLDeltaCRLInd.crl.raw	2011-11-12 04:51:07.448152802 -0500
@@ -6,7 +6,9 @@
         utc "110411185728Z"
         utc "460515185928Z"
         ctx+0x20
-            seq seq oid 2.5.29.20 /* id_cRLNumber */
+            seq seq oid 2.5.29.27 /* id_deltaCRLIndicator */
+                    ocw int 2
+                seq oid 2.5.29.20 /* id_cRLNumber */
                     ocw int 0x01 /* 1 */ 
                 seq oid 2.5.29.35 /* id_authKeyId */
                     ocw seq ctx+0x00
