--- badCMSHasCRL.raw.old	2011-09-22 16:49:17.000000000 -0400
+++ badCMSHasCRL.raw	2011-09-22 16:53:35.000000000 -0400
@@ -98,6 +98,36 @@
                         0xF98EE5DB53F7B5AF71CABA35D78846C4
                         0xEF2646F1AD41E107087C1160A10698C7
                         0x0A
+            ctx+0x21
+                seq seq int 0x01 /* 2 */
+                        seq oid 1.2.840.113549.1.1.11 /* id_sha_256WithRSAEncryption */
+                            nul
+                        seq set seq oid 2.5.4.3 /* id_commonName */
+                                    prt "root"
+                        utc "110910200210Z"
+                        utc "111208200210Z"
+                        seq seq int 0x0D /* 13 */
+                                utc "110218200210Z"
+                            seq int 0x0E /* 14 */
+                                utc "110302200210Z"
+                        ctx+0x20
+                            seq seq oid 2.5.29.20 /* id_cRLNumber */
+                                    ocw int 0x01 /* 1 */
+                                seq oid 2.5.29.35 /* id_authKeyId */
+                                    ocw seq ctx+0x00
+                                                0x6972E4236BB348AA2C9A138434D64D73
+                                                0xDC5A7C69
+                    seq oid 1.2.840.113549.1.1.11 /* id_sha_256WithRSAEncryption */
+                        nul
+                    bit 0x007681C88D4D6451839C75BD3A52D5D2D21CC2E319182A1B2FD32AC181392DF8
+                        0x6A67938FF098B8B0C27A022C7EAA77350760E3874EACBEE30A74D8E7B2E74C69
+                        0x8C6661C0496AE39BF83E7CD4551358F1897A2FBAF2B02D4EC596A39C73B76638
+                        0x6EE4516E2D37A89DCC72FD8AFD56C87A8EB7F3060051742A0DFB82583EB1E726
+                        0x183283D2FB88855FDF0FAAEA257A33D40CC79439115DDEDF16BA1351828F9740
+                        0x498539BCC01296AC989E7676FAF86126E58E82C155422D451FE6E011DA1EBF6A
+                        0xD8FAABD920508BC0908B05DD1C6144C4CD909BBF8A8052F907B3C2A3F8000AEF
+                        0x7347DC4364AD0D8005153DE92E13C8B45022D0FDD24F94E454827CD7F0E1A1D1
+                        0x8C
             set seq int 0x03 /* 3 */ 
                     ctx+0x00
                         0x02859FC8B8CC33675380C96BF01B8B51
