--- badCMSSigInfoAttrsSigTime2Val.raw.old	2013-09-03 10:04:28.147183734 -0400
+++ badCMSSigInfoAttrsSigTime2Val.raw	2013-09-03 10:08:04.285736136 -0400
@@ -116,6 +116,9 @@
                         seq oid 1.2.840.113549.1.9.4 /* id_messageDigestAttr */
                             set oct 0x5808AD5429D14BE2CD5CD9B912B61FD0
                                     0x234CB1D2DD939DB9BC65DAC7014366E4
+                        seq oid 1.2.840.113549.1.9.5 /*id_signingTimeAttr */
+                            set utc "110525220348Z"
+                                utc "110625220348Z"
                     seq oid 1.2.840.113549.1.1.11 /* id_sha_256WithRSAEncryption */
                         nul 
                     oct 0x94A6274EB619F13BF57E515BFD847692
