--- badCertResourcesBadV4Order.raw.old	2011-09-19 16:47:56.000000000 -0400
+++ badCertResourcesBadV4Order.raw	2011-09-19 16:48:47.000000000 -0400
@@ -68,6 +68,7 @@
                     boo 0xFF
                     ocw seq seq oct 0x0001
                                 seq bit 0x000101
+                                    bit 0x000001
                             seq oct 0x0002
                                 seq bit 0x00010201
                 seq oid 1.3.6.1.5.5.7.1.8 /* id_pe_autonomousSysNum */
