--- badCMSSigInfoHashAlg.raw.old	2011-09-22 17:32:24.000000000 -0400
+++ badCMSSigInfoHashAlg.raw	2011-09-22 17:32:44.000000000 -0400
@@ -108,7 +108,7 @@
                     ctx+0x00
                         0xD8E0423673D5DA78450961BF5E068FDC
                         0xF52E0A52
-                    seq oid 2.16.840.1.101.3.4.2.1 /* id_sha256 */
+                    seq oid 2.16.840.1.101.3.4.2.2 /* id_sha256 */
                         nul 
                     ctx+0x20
                         seq oid 1.2.840.113549.1.9.3 /* id_contentTypeAttr */
