--- badCertIssuer2SetComName.raw.old	2013-09-30 19:02:38.054132861 -0400
+++ badCertIssuer2SetComName.raw	2013-09-30 19:03:04.442131758 -0400
@@ -5,6 +5,8 @@
             nul 
         seq set seq oid 2.5.4.3 /* id_commonName */
                     prt "root"
+            set seq oid 2.5.4.3 /* id_commonName */
+                    prt "root"
         seq utc "110411185728Z"
             utc "460515185928Z"
         seq set seq oid 2.5.4.3 /* id_commonName */
