--- badCRLIssDistPt.crl.raw.old	2011-11-12 11:46:22.073670064 -0500
+++ badCRLIssDistPt.crl.raw	2011-11-12 11:47:27.968197249 -0500
@@ -6,7 +6,11 @@
         utc "110411185728Z"
         utc "460515185928Z"
         ctx+0x20
-            seq seq oid 2.5.29.20 /* id_cRLNumber */
+            seq seq oid 2.5.29.28 /* id_issuingDistributionPoint */
+                    ocw seq ctx+0x20
+                                ctx+0x20
+                                    ctx+0x01 "some name"
+                seq oid 2.5.29.20 /* id_cRLNumber */
                     ocw int 0x01 /* 1 */ 
                 seq oid 2.5.29.35 /* id_authKeyId */
                     ocw seq ctx+0x00
