--- badCMSSigInfoWrongSid.raw.old	2011-11-11 08:48:23.410096626 -0500
+++ badCMSSigInfoWrongSid.raw	2011-11-11 08:49:02.250094655 -0500
@@ -105,9 +105,9 @@
                         0xA8A1D4EB5D516CC8DE3B6CB4A81131A8
                         0xFD
             set seq int 0x03 /* 3 */ 
-                    ctx+0x00
-                        0x3BA99190A279CE2AEEA2CF5387EEAE3D
-                        0x621A666E
+                    seq seq set seq oid 2.5.4.3 /* id_commonName */
+                                    prt "root"
+                        int 0x020E /* 526 */
                     seq oid 2.16.840.1.101.3.4.2.1 /* id_sha256 */
                         nul 
                     ctx+0x20
