--- badCMSSigInfoAttrs2BinSigTime.raw.old	2011-11-09 16:29:00.820410581 -0500
+++ badCMSSigInfoAttrs2BinSigTime.raw	2011-11-09 16:29:35.598364954 -0500
@@ -111,6 +111,10 @@
                     seq oid 2.16.840.1.101.3.4.2.1 /* id_sha256 */
                         nul 
                     ctx+0x20
+                        seq oid 1.2.840.113549.1.9.16.2.46 /* binsigtime */
+                            set int 0x0123456789
+                        seq oid 1.2.840.113549.1.9.16.2.46 /* binsigtime */
+                            set int 0x0123456789
                         seq oid 1.2.840.113549.1.9.3 /* id_contentTypeAttr */
                             set oid 1.2.840.113549.1.9.16.1.24 /* id_routeOriginAttestation */
                         seq oid 1.2.840.113549.1.9.4 /* id_messageDigestAttr */
