--- badCMS2DigestAlgs.raw.old	2011-11-11 08:02:31.701077554 -0500
+++ badCMS2DigestAlgs.raw	2011-11-11 08:05:23.859790637 -0500
@@ -2,7 +2,9 @@
     ctx+0x20
         seq int 0x03 /* 3 */ 
             set seq oid 2.16.840.1.101.3.4.2.1 /* id_sha256 */
-                    nul 
+                    nul
+                seq oid 2.16.840.1.101.3.4.2.1 /* id_sha256 */
+                    nul
             seq oid 1.2.840.113549.1.9.16.1.24 /* id_routeOriginAttestation */
                 ctx+0x20
                     ocw seq int 0x08 /* 8 */ 
