--- badCertSIANoRepo.raw.old	2013-08-15 17:09:10.631223755 -0400
+++ badCertSIANoRepo.raw	2013-08-15 17:09:27.974757569 -0400
@@ -53,11 +53,7 @@
                                 ctx+0x06
                                     "rsync://rpki.bbn.com/conformance/root.cer"
                 seq oid 1.3.6.1.5.5.7.1.11 /* id_pe_subjectInfoAccess */
-                    ocw seq seq oid 1.3.6.1.5.5.7.48.5 /* id_ad_caRepository */
-                                ctx+0x06
-                                    "rsync://rpki.bbn.com/conformance/root/empt"
-                                    "y/"
-                            seq oid 1.3.6.1.5.5.7.48.10 /* id_ad_rpkiManifest */
+                    ocw seq seq oid 1.3.6.1.5.5.7.48.10 /* id_ad_rpkiManifest */
                                 ctx+0x06
                                     "rsync://rpki.bbn.com/conformance/root/empt"
                                     "y/doesNotExist.mft"
