--- badGBRIPv4NotInherit.ee.raw.old	2013-05-21 18:34:26.162865209 -0400
+++ badGBRIPv4NotInherit.ee.raw	2013-05-21 18:37:06.446858505 -0400
@@ -60,7 +60,7 @@
                 seq oid 1.3.6.1.5.5.7.1.7 /* id_pe_ipAddrBlock */
                     boo 0xFF
                     ocw seq seq oct 0x0001
-                                nul 
+                                seq bit 0x000101
                             seq oct 0x0002
                                 nul 
                 seq oid 1.3.6.1.5.5.7.1.8 /* id_pe_autonomousSysNum */
