--- badMFTUpdCrossed.raw.old	2013-10-22 10:35:10.844006720 -0400
+++ badMFTUpdCrossed.raw	2013-10-22 10:35:28.980562899 -0400
@@ -7,7 +7,7 @@
                 ctx+0x20
                     ocw seq int 0x01 /* 1 */ 
                             gen "20110411185728Z"
-                            gen "20460515185928Z"
+                            gen "20060515185928Z"
                             oid 2.16.840.1.101.3.4.2.1 /* id_sha256 */
                             seq seq ia5 "MFTUpdCrossed.crl"
                                     bit 0x006885EB00F8368DF6B114947085F9D4
