--- badMFTDuplicateFileOneHash.raw.old	2012-01-25 11:25:58.778996116 -0500
+++ badMFTDuplicateFileOneHash.raw	2012-01-25 11:26:57.539672438 -0500
@@ -13,6 +13,10 @@
                                     bit 0x00B3B4C3D15BF9BF48AA9BAE422CA2BF
                                         0x06C27A3F7E84E298ABEA832A62CAFE63
                                         0xEC
+                                seq ia5 "MFTDuplicateFileOneHash.crl"
+                                    bit 0x00B3B4C3D15BF9BF48AA9BAE422CA2BF
+                                        0x06C27A3F7E84E298ABEA832A62CAFE63
+                                        0xEC
             ctx+0x20
                 seq seq ctx+0x20
                             int 0x02 /* 2 */ 
