--- badROAIPv6OnlyPfxOverlapHighRange.ee.raw.old	2013-10-17 14:16:12.632251542 -0400
+++ badROAIPv6OnlyPfxOverlapHighRange.ee.raw	2013-10-17 14:17:21.949380327 -0400
@@ -59,10 +59,9 @@
                     ocw seq seq oid 1.3.6.1.5.5.7.14.2 /* id_pkix_rescerts_policy */
                 seq oid 1.3.6.1.5.5.7.1.7 /* id_pe_ipAddrBlock */
                     boo 0xFF
-                    ocw seq seq oct 0x0001
-                                seq bit 0x000101
-                            seq oct 0x0002
-                                seq bit 0x00010201
+                    ocw seq seq oct 0x0002
+                                seq seq bit 0x0001020100000000000000000000000001
+                                        bit 0x0001020200000000000000000000000000
                 seq oid 1.3.6.1.5.5.7.1.8 /* id_pe_autonomousSysNum */
                     boo 0xFF
                     ocw seq ctx+0x20
