--- badROAIPv6GoodIPv4Bad.raw.old	2013-10-04 09:38:27.024060397 -0400
+++ badROAIPv6GoodIPv4Bad.raw	2013-10-04 09:38:37.491379194 -0400
@@ -9,7 +9,7 @@
                             seq seq oct 0x0001
                                     seq seq bit 0x000101
                                 seq oct 0x0002
-                                    seq seq bit 0x00010201
+                                    seq seq bit 0x00010203
             ctx+0x20
                 seq seq ctx+0x20
                             int 0x02 /* 2 */ 
