--- badROAIPv6DupPrefixSameMaxLen.raw.old	2013-09-13 14:19:45.032595458 -0400
+++ badROAIPv6DupPrefixSameMaxLen.raw	2013-09-13 14:19:53.325851226 -0400
@@ -10,6 +10,7 @@
                                     seq seq bit 0x000101
                                 seq oct 0x0002
                                     seq seq bit 0x00010201
+                                        seq bit 0x00010201
             ctx+0x20
                 seq seq ctx+0x20
                             int 0x02 /* 2 */ 
