--- badROAIPv6OnlyPrefixSupersetHighPrefix.raw.old	2013-09-27 17:32:21.030461220 -0400
+++ badROAIPv6OnlyPrefixSupersetHighPrefix.raw	2013-09-27 17:32:36.160922504 -0400
@@ -6,10 +6,8 @@
             seq oid 1.2.840.113549.1.9.16.1.24 /* id_routeOriginAttestation */
                 ctx+0x20
                     ocw seq int 0x08 /* 8 */ 
-                            seq seq oct 0x0001
-                                    seq seq bit 0x000101
-                                seq oct 0x0002
-                                    seq seq bit 0x00010201
+                            seq seq oct 0x0002
+                                    seq seq bit 0x01010200
             ctx+0x20
                 seq seq ctx+0x20
                             int 0x02 /* 2 */ 
