--- badROAIPv4ExtraPfxBelowRange.ee.raw.old	2013-10-07 11:15:19.008920893 -0400
+++ badROAIPv4ExtraPfxBelowRange.ee.raw	2013-10-07 11:15:53.457991723 -0400
@@ -60,9 +60,8 @@
                 seq oid 1.3.6.1.5.5.7.1.7 /* id_pe_ipAddrBlock */
                     boo 0xFF
                     ocw seq seq oct 0x0001
-                                seq bit 0x000101
-                            seq oct 0x0002
-                                seq bit 0x00010201
+                                seq seq bit 0x00010101
+                                        bit 0x000101
                 seq oid 1.3.6.1.5.5.7.1.8 /* id_pe_autonomousSysNum */
                     boo 0xFF
                     ocw seq ctx+0x20
