--- badROAIPv6DupPrefixDiffMaxLen.ee.raw.old	2013-09-13 17:19:51.581150423 -0400
+++ badROAIPv6DupPrefixDiffMaxLen.ee.raw	2013-09-13 17:20:06.725602511 -0400
@@ -63,6 +63,7 @@
                                 seq bit 0x000101
                             seq oct 0x0002
                                 seq bit 0x00010201
+                                    bit 0x000102011
                 seq oid 1.3.6.1.5.5.7.1.8 /* id_pe_autonomousSysNum */
                     boo 0xFF
                     ocw seq ctx+0x20
