--- badROAIPv4ExtraPfxBelowRange.raw.old	2013-10-07 11:15:53.612995833 -0400
+++ badROAIPv4ExtraPfxBelowRange.raw	2013-10-07 11:16:20.550833365 -0400
@@ -7,9 +7,8 @@
                 ctx+0x20
                     ocw seq int 0x08 /* 8 */ 
                             seq seq oct 0x0001
-                                    seq seq bit 0x000101
-                                seq oct 0x0002
-                                    seq seq bit 0x00010201
+                                    seq seq bit 0x00010101
+                                        seq bit 0x000100
             ctx+0x20
                 seq seq ctx+0x20
                             int 0x02 /* 2 */ 
