--- goodROAASIDZero.raw.old	2013-03-09 19:44:55.945018145 -0500
+++ goodROAASIDZero.raw	2013-03-09 19:45:18.065018119 -0500
@@ -5,7 +5,7 @@
                     nul 
             seq oid 1.2.840.113549.1.9.16.1.24 /* id_routeOriginAttestation */
                 ctx+0x20
-                    ocw seq int 0x08 /* 8 */ 
+                    ocw seq int 0x00 /* 0 */
                             seq seq oct 0x0001
                                     seq seq bit 0x000101
                                 seq oct 0x0002
