--- goodMFTNumMax.raw.old	2012-07-06 11:35:21.321542352 -0400
+++ goodMFTNumMax.raw	2012-07-06 11:37:19.909220040 -0400
@@ -5,7 +5,7 @@
                     nul 
             seq oid 1.2.840.113549.1.9.16.1.26
                 ctx+0x20
-                    ocw seq int 0x01 /* 1 */ 
+                    ocw seq int 0x7FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
                             gen "20110411185728Z"
                             gen "20460515185928Z"
                             oid 2.16.840.1.101.3.4.2.1
