--- goodROAIPv6PfxesEqualRange.raw.old	2013-10-17 14:34:15.547499275 -0400
+++ goodROAIPv6PfxesEqualRange.raw	2013-10-17 14:35:38.946062756 -0400
@@ -6,10 +6,9 @@
             seq oid 1.2.840.113549.1.9.16.1.24 /* id_routeOriginAttestation */
                 ctx+0x20
                     ocw seq int 0x08 /* 8 */ 
-                            seq seq oct 0x0001
-                                    seq seq bit 0x000101
-                                seq oct 0x0002
-                                    seq seq bit 0x00010201
+                            seq seq oct 0x0002
+                                    seq seq bit 0x0101020100000000000000000000000000
+                                        seq bit 0x0001020100000000000000000000000002
             ctx+0x20
                 seq seq ctx+0x20
                             int 0x02 /* 2 */ 
