--- goodROAIPv4OnlyPfxInRangeHigh.raw.old	2013-10-08 13:53:18.466223471 -0400
+++ goodROAIPv4OnlyPfxInRangeHigh.raw	2013-10-08 13:53:32.057638352 -0400
@@ -7,9 +7,7 @@
                 ctx+0x20
                     ocw seq int 0x08 /* 8 */ 
                             seq seq oct 0x0001
-                                    seq seq bit 0x000101
-                                seq oct 0x0002
-                                    seq seq bit 0x00010201
+                                    seq seq bit 0x00010123
             ctx+0x20
                 seq seq ctx+0x20
                             int 0x02 /* 2 */ 
