--- NAMSeqSerName.raw.old	2013-10-01 16:31:25.386898811 -0400
+++ NAMSeqSerName.raw	2013-10-01 16:32:58.334894924 -0400
@@ -7,7 +7,9 @@
                     prt "root"
         seq utc "110411185728Z"
             utc "460515185928Z"
-        seq set seq oid 2.5.4.3 /* id_commonName */
+        seq set seq oid 2.5.4.5
+                    prt "17"
+            set seq oid 2.5.4.3 /* id_commonName */
                     prt "NAMSeqSerName"
         seq seq oid 1.2.840.113549.1.1.1 /* id_rsadsi_rsaEncryption */
                 nul 
