--- NAMSeqNameSer.raw.old	2013-10-01 16:25:29.622913689 -0400
+++ NAMSeqNameSer.raw	2013-10-01 16:30:42.446900606 -0400
@@ -9,6 +9,8 @@
             utc "460515185928Z"
         seq set seq oid 2.5.4.3 /* id_commonName */
                     prt "NAMSeqNameSer"
+            set seq oid 2.5.4.5
+                    prt "42"
         seq seq oid 1.2.840.113549.1.1.1 /* id_rsadsi_rsaEncryption */
                 nul 
             bit 0x003082010A0282010100DB23D121A60B
